scripts/split-e2e-tests-codebuild.ts

```diff
@@ -46,6 +46,10 @@ const DISABLE_COVERAGE = ['src/__tests__/datastore-modelgen.test.ts', 'src/__tes
 const TEST_EXCLUSIONS: { l: string[]; w: string[] } = {
   l: [],
   w: [
+    /* TEMPORARY-PR12830: Remove after we ship PR12830 */
+    'src/__tests__/custom_resources.test.ts',
+    'src/__tests__/custom-resource-with-storage.test.ts',
+    /* END TEMPORARY */
```

Comment:
What's happening here?

Reply:
adding missing piece of https://github.com/aws-amplify/amplify-cli/pull/12830/commits/eba5625eec899a803bf908f5116effc8d50be777 .
I.e. we disabled these on CCI but not CB.